test/es-module/test-loader-worker.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+require('../common');
+const { spawnSync } = require('child_process');
+const tmpdir = require('../common/tmpdir');
+const assert = require('assert');
+const { execPath } = require('process');
+const { writeFileSync } = require('fs');
+const { join } = require('path');
+
+tmpdir.refresh();
+writeFileSync(join(tmpdir.path, 'hello.cjs'), 'console.log("hello")', 'utf8');
+writeFileSync(join(tmpdir.path, 'loader.mjs'), 'import "./hello.cjs"', 'utf8');
```

Comment:
This is necessary for me to reproduce the bug locally - the translator for commonjs modules would invoke the cascaded loader.

Reply:
Does https://github.com/nodejs/node/pull/47599/files#diff-82a900b610fadd4d760c0af4f8707a3477aa9e28ebe03bfcbb2fcc0cd51947cf not reproduce the bug for you? If it does, I'd prefer we use that as it seems simpler and doesn't require to write files to disk, but no strong feelings.

Reply:
Surely we already have fixtures for printing test strings. Can we use existing fixtures rather than create new ones? And I feel like we certainly shouldn’t need to create new files that we dynamically write to disk.